tests/unit/file-test.js

```diff
@@ -0,0 +1,32 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { upload as uploadHandler } from 'ember-file-upload/mirage';
+import File from 'ember-file-upload/file';
+
+module('Unit | File', function(hooks) {
+  setupTest(hooks);
+  setupMirage(hooks);
+
+  test('it can upload without a `queue`', async function (assert) {
+    assert.expect(1);
```

Comment:
Personally I prefer `assert.step()` and [`assert.verifyStep()`](https://api.qunitjs.com/assert/verifySteps) as it's easier to maintain when additional assertions are added. But that's more a question of coding style. `assert.expect()` works as well.